doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Why not have a paragraph here about these computations? How they are done along with some plot of something. It would be good to understand the models more and to see what they can do. We can generate a small trajectory, for example, and have them plot something about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>